src/amo/components/Search/index.js

```diff
@@ -120,6 +124,18 @@ export class SearchBase extends React.Component {
       <SearchSort filters={filters} pathname={pathname} />
     ) : null;
 
+    if (pathname === '/search/' && (!filters.query || filters.query.length === 0)) {
```

Comment:
While this approach will technically work it's fragile (what if we add another search URL or change it?) and it also seems to be untested.

The best approach would be to have this check in the `SearchPage` component because it's the only place we currently care about having a search query.